sci-geosciences/gpxsee/gpxsee-6.0.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="GPS log file viewer and analyzer"
+DESCRIPTION="GPXSee is a Qt-based GPS log file viewer and analyzer that supports \
+GPX, TCX, KML, FIT, IGC, NMEA, SLF, LOC and OziExplorer files."
+HOMEPAGE="http://www.gpxsee.org/"
+COPYRIGHT="2018 Martin Tůma"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/tumic0/gpxsee/archive/$portVersion/GPXSee-$portVersion.tar.gz"
+CHECKSUM_SHA256="250e316fc40e542397720990b943c43cf1725a7b2d665ccf911c9e62d4a8f249"
+SOURCE_DIR="GPXSee-$portVersion"
+PATCHES="gpxsee-$portVersion.patchset"
+ADDITIONAL_FILES="gpxsee.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gpxsee$secondaryArchSuffix = $portVersion
+	app:GPXSee = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
```

Comment:
You can drop curly brackets here, it is mostly needed for haiku_devel package.

Reply:
Updated.